src/components/book.js

```diff
@@ -14,32 +14,51 @@ const Book = (props) => {
   } = props;
   return (
     <div className="book-card">
-      <div className="book-category">
-        <span>Action</span>
-      </div>
-      <div className="book-title">
-        <span>{title}</span>
-      </div>
-      <div className="book-author">
-        <span>{author}</span>
-      </div>
-      <div className="reaction">
-        <ul>
+      <div className="bookDetail">
+        <div className="book-category">
+          <span>Action</span>
+        </div>
+        <div className="book-title">
+          <span>{title}</span>
+        </div>
+        <div className="book-author">
+          <span>{author}</span>
+        </div>
+        <ul className="reaction">
           <li>
-            <button type="button" id={key}>
+            <button type="button" className="left buttons" id={key}>
               Comments
             </button>
           </li>
           <li>
             {' '}
-            <button type="button" id={id} onClick={handleRemove}>Remove</button>
+            <button
+              type="button"
+              id={id}
+              className="left buttons"
+              onClick={handleRemove}
+            >
+              Remove
+            </button>
           </li>
           <li>
             {' '}
-            <button type="button">edit</button>
+            <button className="buttons" type="button">edit</button>
           </li>
         </ul>
```

Comment:
[REQUIRED] Kindly make the dark line between comments, edit and remove more faint.

[REQUIRED] kindy ensure that the button container is well aligned on the left just like the author and title above it.

|Your Design|
|------------|
|![image](https://user-images.githubusercontent.com/62938735/191815316-d85d1c80-7ef1-4942-9f7b-a59b6764308c.png)|